templates/cis2-nft/cargo-generate.toml

```diff
@@ -0,0 +1,7 @@
+[template]
+cargo_generate_version = "0.16.0"
+
+[placeholders]
+description = { type="string", prompt="Description for the crate?" }
+currentAuthors = { type="string", prompt="Authors for this crate?" }
+TOKEN_METADATA_BASE_URL = { type="string", prompt="Token metadata base URL of the token?", default='https://some.example/token/' }
```

Comment:
Why is this in upper-case when the other placeholders are lower-cased?

Reply:
The variable is `const` in the Rust smart contract code and for such variables, it is a convention to use screaming_snake_case (I just copied the variable from the smart contract code here that is how it ended up like this). 

Yes, I agree that in the above code, it would be more consistent if we change the naming to the same style then the other variables. Changed it.